src/index.js

```diff
@@ -93,6 +94,13 @@ async function writePackageJSON() {
   await writeJsonFile(path, pkg, { detectIndent: true });
 }
 
+async function installDeps() {
+  const useNPM = fs.existsSync('package-lock.json');
+  const command = useNPM ? 'npm' : 'yarn';
+  const args = ['install', '--ignore-scripts'];
```

Comment:
Do we need the `--ignore-scripts` option here? I added it because installing via `npm`/`yarn` can sometimes lead into running tests, linting, etc., and idk if we wanted `babel-upgrade` to be as fast as it could be?

Reply:
Not sure if it matters or not? I mean we want the upgrade to pass tests anyway?

Reply:
Right, that makes sense. Will remove!

Reply:
I guess if they use babel you'd have to compile or run tests to verify it works anyway but maybe we don't need to do that for them.

Reply:
I am confused by this TODO: **Run npm/yarn after updating dependencies (use --install) (#18)**
can you give me specifically what command or script to put under npm scripts here. I have no idea yet where --install flag goes not using yarn at all here. hmm it worked when I type "npm install" without this -- not sure what the -- is for?? seems like for under scripts in package.json??
package.json ::

{
  "name": "tutorials",
  "version": "1.0.0",
  "description": "",
  "main": "index.js",
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1"
  },
  "keywords": [],
  "author": "",
  "license": "ISC",
  "devDependencies": {
    "@babel/core": "^7.2.2",
    "@babel/plugin-proposal-class-properties": "^7.0.0",
    "@babel/plugin-proposal-decorators": "^7.0.0",
    "@babel/plugin-proposal-export-namespace-from": "^7.0.0",
    "@babel/plugin-proposal-function-sent": "^7.0.0",
    "@babel/plugin-proposal-json-strings": "^7.0.0",
    "@babel/plugin-proposal-numeric-separator": "^7.0.0",
    "@babel/plugin-proposal-throw-expressions": "^7.0.0",
    "@babel/plugin-syntax-dynamic-import": "^7.0.0",
    "@babel/plugin-syntax-import-meta": "^7.0.0",
    "@babel/preset-env": "^7.3.1",
    "@babel/preset-react": "^7.0.0",
    "babel-loader": "^8.0.5",
    "html-webpack-plugin": "^3.2.0",
    "webpack": "^4.29.3",
    "webpack-cli": "^3.2.3",
    "webpack-dev-server": "^3.1.14"
  }
}